plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/extractor/Developer.java

```diff
@@ -0,0 +1,14 @@
+package io.jenkins.tools.pluginmodernizer.core.extractor;
+
+public record Developer(String id, String name, String email, String role) {
```

Comment:
Developer section are not used anymore on Jenkins pom.xml.

In fact that even could be a receipt to cleanup such pom